scripts/packaging/linux_distro.py

```diff
@@ -193,7 +193,7 @@ def create_debian_bundle(self):
                 "libglib2.0-0 (>= 2.10.0), libegl1, libglu1-mesa | libglu1, libgtk-3-0, "
                 "libnss3, libstdc++6 (>= 4.0.2-4), libxaw7, libxrandr2, libxrender1, "
                 "libssh-dev, libzip-dev, xserver-xorg-core, libxslt1.1, "
-                "libxerces-c-dev, libfox-1.6-dev, libgdal-dev, libproj-dev, libgl2ps-dev, "  # SUMO dependencies
+                "libxerces-c3.2, libfox-1.6-0, libgdal-dev, libproj-dev, libgl2ps1.4, "  # SUMO dependencies
```

Comment:
I kept `libgdal-dev`and `libproj-dev` just because the version changed in Ubuntu 22.04 and this way the version is managed automatically by the packaging tool.